api/HTMLFormElement.json

```diff
@@ -599,10 +599,10 @@
               "version_added": null
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
```

Comment:
This entry, `noValidate`, was also not updated in https://github.com/mdn/browser-compat-data/pull/7056.